stdlib/mexpr/pprint.mc

```diff
@@ -657,7 +662,7 @@ lang SeqEdgePatPrettyPrint = SeqEdgePat + PatNamePrettyPrint
   sem getPatStringCode (indent : Int) (env : PprintEnv) =
   | PSeqEdge {prefix = pre, middle = patname, postfix = post} ->
     match _pprint_patseq getPatStringCode indent env pre with (env, pre) then
-    match _pprint_patname env patname with (env, pname) then
+    match getPatStringCode indent env patname with (env, pname) then
```

Comment:
Also related to the comment on `PatName` vs `Pat`, the middle should just be a `PatName`, not a full `Pat`, so the previous implementation was correct. I guess the `pprint` "tests" don't contain an edge pattern, so this doesn't explode.

Reply:
The problem has been solved.